control-plane/main.tf

```diff
@@ -202,6 +203,25 @@ module "install_common" {
   ]
 }
 
+resource "null_resource" "set_loglevel" {
+  count = var.rancher_loglevel != "info" ? 1 : 0
+  provisioner "local-exec" {
+    interpreter = [
+      "bash", "-c"
+    ]
+    command = <<-EOT
+    kubectl --kubeconfig <(echo $KUBECONFIG | base64 --decode) -n cattle-system get pods -l app=rancher --no-headers -o custom-columns=name:.metadata.name | while read rancherpod; do kubectl --kubeconfig <(echo $KUBECONFIG | base64 --decode) -n cattle-system exec $rancherpod -c rancher -- loglevel --set ${var.rancher_loglevel}; done
```

Comment:
Do we ever do any single node Rancher deployments in scaling ? I am assuming no but just double checking

Reply:
Nope! Its never come up as something that needs scale testing